docs/_quarto.yml

```diff
@@ -25,11 +24,45 @@ metadata-files:
   - api/_sidebar.yml
 
 number-sections: false
+quartodoc:
+  dir: api_ref
+  # dynamic: true
+  sections:
+    - title: flood_adapt
+      desc: ""
+      contents:
+        - name: api.benefits
+        - name: api.database_builder
+        - name: api.events
+        - name: api.measures
+        - name: api.output
+        - name: api.projections
+        - name: api.scenarios
+        - name: api.static
+        - name: api.strategies
+        - name: adapter.fiat_adapter
+        - name: adapter.sfincs_adapter
+        - name: dbs_classes.dbs_benefit.DbsBenefit
+        - name: dbs_classes.dbs_event.DbsEvent
+        - name: dbs_classes.dbs_measure.DbsMeasure
+        - name: dbs_classes.dbs_projection.DbsProjection
+        - name: dbs_classes.dbs_scenario.DbsScenario
+        - name: dbs_classes.dbs_static.DbsStatic
+        - name: dbs_classes.dbs_strategy.DbsStrategy
+        - name: dbs_classes.dbs_template.DbsTemplate
+        - name: dbs_controller.Database
+        - name: object_model.benefit.Benefit
+        - name: object_model.direct_impacts.DirectImpacts
+        - name: object_model.scenario.Scenario
+        - name: object_model.site.Site
+        - name: object_model.strategy.Strategy
+        - name: FloodAdaptLogging
+  package: flood_adapt
```

Comment:
Is there a way this can be generated from source code? 
This is 100% something people will forget to update if they add new files / classes

Reply:
in theory? yes. As is? no. This is also to answer the question you put in your review comment. This is not what it does exactly, but what it seems `quartodoc` does to build it's links is `from <package> import *` to figure out what to document. Usually this works quite well since submodules usually re-export their public parts so by just importing everything from the root, you can get the important things. The reason that this doesn't work for us, is that we don't import our submodules from the root, presumably because of circular imports which I ran into immediately when I tried that. This was the best compromise I could find

Reply:
Yeah I remember there being problems with that. Ok it is what it is then. Also since its the api, these things shouldnt change that often